frontend/src/pages/auth/Register.tsx

```diff
@@ -0,0 +1 @@
+export {}
```

Comment:
Delete the file if it doesn't containt anything

Reply:
Done in 297b95af686333995dfbb1642990bc3e4eced889